src/HPolyhedron.jl

```diff
@@ -67,6 +67,34 @@ function dim(P::HPoly{N})::Int where {N<:Real}
     return length(P.constraints) == 0 ? -1 : length(P.constraints[1].a)
 end
 
+"""
+    ρ(d::AbstractVector{N}, P::HPoly{N})::N where {N<:Real}
+
+Evaluate the support function of a polyhedron (in H-representation) in a given
+direction.
+
+### Input
+
+- `d` -- direction
+- `P` -- polyhedron in H-representation
+
+### Output
+
+The support function of the polyhedron.
+If the polyhedron is unbounded in the given direction, the result is `Inf`.
+
+### Algorithm
+
+This implementation uses `GLPKSolverLP` as linear programming backend.
+"""
+function ρ(d::AbstractVector{N}, P::HPoly{N})::N where {N<:Real}
```

Comment:
use `HPolyhedron` instead of `HPoly`?

Reply:
on the other hand we dont yet a have a `isbounded` check. . i take back my comment.

Reply:
Oh, I forgot to make the result depend on the type :+1: